docs/how-to-configure-new-repository.md

```diff
@@ -14,18 +14,24 @@ Documents [Community Membership](../community-membership.md) and
 [CONTRIBUTING](../CONTRIBUTING.md) define how permissions are typically set up
 for the repository.
 
-1. Every repository has two teams associated with it. Typically they will be
-   named `foo-approvers` and `foo-maintainers`. `foo-maintainers` is a child of
-   `foo-approvers` as it always contains subset of people and defines larger
-   scope of privileges.
-2. Both teams has `Write` permissions for the repository.
-3. Root-level `CODEOWNERS` file on the repository should include superset of
+1. Every repository has two teams associated with it. Typically for the
+   repository `opentelemetry-foo` they will be named `foo-approvers` and
+   `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers` as it
+   always contains subset of people and defines larger scope of privileges.
+2. Event though the members of `foo-maintainers` are the members of
+   `foo-approvers` automatically as the groups are enclosed, every member of
+   `foo-maintainers` should be included in `foo-approvers` explicitly with the
+   "Maintainer". So repository maintainers can invite new approvers.
```

Comment:
what is `the "Maintainer"`?

Reply:
you mean `the "Maintain" GitHub permissions`???

Reply:
yes, the new Maintain github permissions, I guess.

Reply:
I mean team maintainer role. So maintainers will be able to add approvers

![image](https://user-images.githubusercontent.com/9950081/65219739-08c69300-da6e-11e9-9aa3-11bf9248f27c.png)

Reply:
Gotcha. Let's say

```
Even though the members of `foo-maintainers` are included in `foo-approvers` transitively, every member of `foo-maintainers` should be included in `foo-approvers` explicitly, and with the "Maintainer" GitHub privileges.
```

Reply:
fixed